api/app/tests/weather_models/test_grib_processing.feature

```diff
@@ -54,3 +54,18 @@ Feature: Grib file processing
             | [-180.075, 0.15000000000000002, 0.0, 90.075, 0.0, -0.15]            | CMC_glb_latlon.15x.15_projection_wkt.txt         | [370, 330]        | [-124.57499999999999, 40.575]             |
             | [-180.075, 0.15000000000000002, 0.0, 90.075, 0.0, -0.15]            | CMC_glb_latlon.15x.15_projection_wkt.txt         | [315, 455]        | [-132.825, 21.825000000000003]            |
             | [-180.075, 0.15000000000000002, 0.0, 90.075, 0.0, -0.15]            | CMC_glb_latlon.15x.15_projection_wkt.txt         | [427, 245]        | [-116.02499999999998, 53.325]             |
+
+
+    Scenario: Calculate wind speed and direction from U,V components
```

Comment:
I'd highly, highly, recommend we avoid adding any new gherkin tests because:
1) It's buggy: https://github.com/bcgov/wps/pull/2288#issuecomment-1255242576
2) The version we're on depends on an old dependency causing a security alert, so we have a ticket to remove it: https://app.zenhub.com/workspaces/bcws---agile-psu-5e321393e038fba5bbe203b8/issues/gh/bcgov/wps/2561, this would mean having to painstakingly refactor the tests that Sy tried to do before and found issues; it's a huge maintenance burden.
3) Alternatives should be easier for you to implement in terms of effort, or at least, effort checking that tests are working

I'd propose we just use the built in `TestClient` that FastAPI supplies, and supply input/expected output as JSON  for complex data, or hand crafted request/response schema models for simpler cases, take a look at `api/app/tests/hfi/test_hfi_admin_endpoint.py` for some examples. Also happy to discuss further.